docs/pages/admin-guides/teleport-policy/integrations/crown-jewels.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: See permission changes with Teleport Policy Crown Jewels
+description: Describes how to use Teleport Policy Crown Jewels to see permission changes in Teleport.
+---
+
+With Teleport Policy's Access Graph, you can see change to users and resources permissions.
+Crown Jewels is a feature that allows you to mark resources and users as critical and see changes to them.
+Each time access for those resources or users changes, Teleport Policy will log the change and generate
+before and after snapshots of the permissions. This can help you identify when users get access to sensitive
+resources or when permissions are changed unexpectedly.
+
+This guide shows you how to configure Crown Jewels, how to mark resource and how to see permission changes.
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v16.2.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- For self-hosted clusters, a running Access Graph node v1.24.0 or later.
+Check [Access Graph page](../teleport-policy.mdx) for details on
+how to set up Access Graph.
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise edition customers.
+
+After logging in to the Teleport UI, navigate to the Management tab. If enabled, Access Graph options can be found
+under the Permission Management section.
```

Comment:
Note that this will need to be changed soon because Teleport 17 won't have a management tab.

If we can, it would be best to keep this description as generic as possible so it doesn't need to be rewritten if the UI changes.